### ISSUE TITLE
csp_whitelist.xml fix for https://static-eu.payments-amazon.com - img-src asset

### ISSUE THREAD

**Issue by daniel-ifrim** [amzn/amazon-payments-magento-2-plugin#1057]
Report of blocked resource in browser:

#### Your setup

* Magento version: 2.4.2
* Amazon Pay Extension Version: 5.5.0
* Magento Edition: Community
* Amazon Germany

```
[Report Only] Refused to load the image 'https://static-eu.payments-amazon.com/assets/maxo/microtext/en_GB.svg' because it violates the following Content Security Policy directive: "img-src assets.adobedtm.com amcglobal.sc.omtrdc.net dpm.demdex.net cm.everesttech.net widgets.magentocommerce.com data: www.googleadservices.com www.google-analytics.com www.paypalobjects.com t.paypal.com *.ftcdn.net *.behance.net www.paypal.com fpdbs.paypal.com fpdbs.sandbox.paypal.com *.vimeocdn.com s.ytimg.com d3sbl0c71oxeok.cloudfront.net dhkkzdfmpzvap.cloudfront.net d2bpzs5y44q6e0.cloudfront.net d37shgu97oizpd.cloudfront.net d1zlqll3enr74n.cloudfront.net d1jynp0fpwn93a.cloudfront.net d2cb3tokgpwh3v.cloudfront.net d1re8bfxx3pw6e.cloudfront.net d35u8xwkxs8vpe.cloudfront.net d13s9xffygp5o.cloudfront.net d388nbw0dwi1jm.cloudfront.net d11p2vtu3dppaw.cloudfront.net d3r89hiip86hka.cloudfront.net dc7snq0c8ipyk.cloudfront.net d5c7kvljggzso.cloudfront.net d2h8yg3ypfzua1.cloudfront.net d1b556x7apj5fb.cloudfront.net draz1ib3z71v2.cloudfront.net dr6hdp4s5yzfc.cloudfront.net d2bomicxw8p7ii.cloudfront.net d3aypcdgvjnnam.cloudfront.net d2a3iuf10348gy.cloudfront.net d23yuld0pofhhw.cloudfront.net *.ssl-images-amazon.com *.ssl-images-amazon.co.uk *.ssl-images-amazon.co.jp *.ssl-images-amazon.jp *.ssl-images-amazon.it *.ssl-images-amazon.fr *.ssl-images-amazon.es *.ssl-images-amazon.de *.media-amazon.com *.media-amazon.co.uk *.media-amazon.co.jp *.media-amazon.jp *.media-amazon.it *.media-amazon.fr *.media-amazon.es *.media-amazon.de maps.gstatic.com/ *.yotpo.com 'self' 'unsafe-inline'".
```

Thank you

**Comment by jaybeckr** [comment]
@daniel-ifrim - thanks for the report, we'll include a fix in the next release. You can apply the patch from https://github.com/BearGroup/amazon-payments-magento-2-plugin/commit/dcb25483ef224f5af095ba0d7f5b88d6808f64d3 if you need it sooner.

**Comment by christianzichichi** [comment]
Hi @daniel-ifrim , this has been fixed in the newest 5.5.1 release

Thanks
Best

Christian
